docs/05-technical-reference/00-architecture/smgt-10-hb.md

```diff
@@ -4,7 +4,7 @@ title: Helm Broker basic architecture
 
 Helm Broker is installed alongside other Kyma components and it automatically registers itself in Service Catalog as a ClusterServiceBroker. The installation provides the default [helm-repos-urls](https://github.com/kyma-project/kyma/blob/main/resources/helm-broker/templates/addons-cfg.yaml) ClusterAddonsConfiguration (CAC) custom resource (CR). It contains URLs from which Helm Broker fetches addons. You can also add your own addons with URLs that point to [your addons repository](../../03-tutorials/00-service-management/smgt-14-hb-create-addons-repo.md).
 
-If you want Helm Broker to act as a Namespace-scoped ServiceBroker, create the [AddonsConfiguration](../06-custom-resources/smgt-04-hb-addonsconfiguration.md) (AC) CR. In such a case, Helm Broker creates a service and registers itself in Service Catalog as a ServiceBroker inside the Namespace in which the CR is created.
+If you want Helm Broker to act as a Namespace-scoped ServiceBroker, create the [AddonsConfiguration](../00-custom-resources/smgt-04-hb-addonsconfiguration.md) (AC) CR. In such a case, Helm Broker creates a service and registers itself in Service Catalog as a ServiceBroker inside the Namespace in which the CR is created.
```

Comment:
```suggestion
If you want Helm Broker to act as a Namespace-scoped ServiceBroker, create the [AddonsConfiguration](../00-custom-resources/smgt-04-hb-addonsconfiguration.md) CR. In such a case, Helm Broker creates a service and registers itself in Service Catalog as a ServiceBroker inside the Namespace in which the CR is created.
```

Are you sure you want to use the AC acronym for this? We've been using AC to mean Application Connector/Application Connectivity for years, so this might be confusing. Especially if someone starts using this acronym outside of this document. I'd strongly advise against this.

Reply:
They are going to be removed with Service Management 2.0 anyway, so we won't face this issue anymore 🙂